app/src/componentLibrary/autocompletes/singleAutocomplete/singleAutocomplete.jsx

```diff
@@ -51,6 +51,7 @@ export class SingleAutocomplete extends Component {
     icon: PropTypes.string,
     isOptionUnique: PropTypes.func,
     refFunction: PropTypes.func,
+    prohibitCreateOnBlur: PropTypes.bool,
```

Comment:
If you need to prevent custom option creation at all, it is better to introduce the `creatable` prop as it was implemented in the old autocomplete version. Also I see the same feature inside `MultipleAutocomplete`.